devicetypes/aamann/enlighten-envoy-local.src/enlighten-envoy-local.groovy

```diff
@@ -296,9 +302,10 @@ def ping() {
 
 def startPoll() {
 	unschedule()
-	// Schedule 2 minute polling
+	def wantMin = 6       // Schedule N minute polling Integer < 60
```

Comment:
I guess the best way to go about changing the polling interval would be to expose it as a setting rather than hard coding.